plotting/timeseries.py

```diff
@@ -137,34 +158,28 @@ def csv(self):
             "Longitude",
             "Time",
         ]
+        
+        # Check to see if the quiver attribute is present. If so the CSV export will 
+        # also include X and Y velocity components (pulled from the quiver_data attribute) 
+        # and bearing information (to be calculated below).
+        have_quiver = hasattr(self, 'quiver_data')
 
-        if len(self.variables) > 1:
-            # Under the current API this indicates that velocity has been
-            # selected, which actually is derived from two variables: X and Y
-            # velocities. As such, the CSV export will also include X and Y
-            # velocity components (pulled from the quiver_data attribute) and
-            # bearing information (to be calculated below).
-            have_quiver = hasattr(self, 'quiver_data')
-        else:
-            have_quiver = False
 
         if self.depth != 'all':
             if isinstance(self.depth, str) or isinstance(self.depth, str):
                 header.append(["Depth", self.depth])
             else:
                 header.append(
-                    ["Depth", "%.4f%s" % (self.depths,
+                    ["Depth", "%.4f%s" % (self.depth,
```

Comment:
By changing this to `depth`, how would this affect a timeseries CSV over all depths? See https://github.com/DFO-Ocean-Navigator/Ocean-Data-Map-Project/blob/master/tests/test_api_v_1_0.py#L296

Reply:
DOes it make sense to create a CSV for the whole water column?